doc/neps/nep-0005-generalized-ufuncs.rst

```diff
@@ -1,7 +1,7 @@
 .. _NEP05:
 
 =======================================
-NEP 5 — Generalized Universal Functions
+NEP 5 — Generalized universal functions
```

Comment:
I have a feeling that Universal Functions is a "proper name" and should be capitalized, but I'd defer to a native English speaker to confirm...

Reply:
I ran a spaCy model to confirm (English - en_core_web_sm (v2.3.0). Its a Proper noun. I shall make  the changes
![Screenshot 2021-09-08 at 8 22 57 PM](https://user-images.githubusercontent.com/17350312/132533511-52b3c326-ccab-41d3-a3d9-06b7eb5c3d18.png)